tests/resource_filtering_test.go

```diff
@@ -2808,6 +2807,61 @@ var _ = Describe("Resource excludes", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
+			It("Negative: VM+VMI+Pod included should fail if VM is paused", func() {
+				By("Creating VirtualMachines")
+				vmSpec := framework.CreateVmWithGuestAgent("test-vm", r.StorageClass)
+				vmIncluded, err := framework.CreateVirtualMachineFromDefinition(kvClient, namespace.Name, vmSpec)
+				Expect(err).ToNot(HaveOccurred())
+				err = framework.WaitForDataVolumePhase(kvClient, namespace.Name, cdiv1.Succeeded, vmSpec.Spec.DataVolumeTemplates[0].Name)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Starting the virtual machine")
+				err = framework.StartVirtualMachine(kvClient, namespace.Name, vmSpec.Name)
+				Expect(err).ToNot(HaveOccurred())
+				err = framework.WaitForVirtualMachineStatus(kvClient, namespace.Name, vmSpec.Name, kvv1.VirtualMachineStatusRunning)
+				Expect(err).ToNot(HaveOccurred())
+				ok, err := framework.WaitForVirtualMachineInstanceCondition(kvClient, namespace.Name, vmSpec.Name, kvv1.VirtualMachineInstanceAgentConnected)
```

Comment:
Since we are pausing the VM anyway, do we need to bother waiting for the agent to connect?  Might be able to speed up this test quite a bit.

Reply:
right, I'll try this

Reply:
@brybacki good point. If it's necessary for the test then everything looks good to me.